Assets/StreamingAssets/Quests/_TUTOR__.txt

```diff
@@ -9,388 +9,355 @@ Quest: _TUTOR__
 QRC:
 
 RumorsDuringQuest:  [1005]
-<ce>                       I'll give you a hint %ra.
-<ce>                   If you want to succeed in magic,
-<ce>                         join the mages guild.
+I'll give you a hint %ra.  If you want to succeed in magic, join the mages guild.
```

Comment:
You got a few double spaces in there. I'd do a pass with Find "  " and replace it with " "